src/commands/down.ts

```diff
@@ -0,0 +1,39 @@
+import { Command, flags } from "@oclif/command";
+
+import chalk from "chalk";
+import * as shelljs from "shelljs";
+import { testTargetDirectory, displayCommandHeader } from "../actions";
+
+export default class Down extends Command {
+  static description = "describe the command here";
+
+  static flags = {
+    verbose: flags.boolean({ char: "v", default: false })
+  };
+
+  static args = [
+    {
+      name: "directory",
+      description: "Running Laravel Up directory you would like to stop"
+    }
+  ];
+
+  async run() {
+    const { args, flags } = this.parse(Down);
+
+    displayCommandHeader("Shutting down your project...");
+
+    const directory = args.directory
+      ? args.directory
+      : "/home/jesse/code/a-test-laravel";
```

Comment:
It should default to the current working directory instead of a hard coded path. You can get it with `shelljs.pwd().toString()`